<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>studentMenu</class>
 <widget class="QWidget" name="studentMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>137</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Professor Menu</string>
  </property>
  <widget class="QPushButton" name="studentexamsButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Exams</string>
   </property>
  </widget>
  <widget class="QPushButton" name="studentgradesButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Grades</string>
   </property>
  </widget>
  <widget class="QLabel" name="professprmenuLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Student Menu:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>studentexamsButton</sender>
   <signal>clicked()</signal>
   <receiver>studentMenu</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>68</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>studentgradesButton</sender>
   <signal>clicked()</signal>
   <receiver>studentMenu</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>68</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
